nemo_skills/inference/generate.py

```diff
@@ -105,9 +105,7 @@ class GenerateSolutionsConfig:
     # and so on
     multi_turn_key: str | None = None
 
-    # set to False if you want to use synchronous loop instead of async. Async loop means we will send all
-    # data to engine at the same time (batch size is ignored) and then write the output as soon as it's ready
-    # to `output_file`-async (and put it back in order after all generations are done)
+    # Must always be True, sync is no longer supported
     use_async_loop: bool = True
```

Comment:
Should we remove this parameter completely then?

Reply:
Well, it defaulted to True, so it's only really meaningful to set it to False, but that's not supported anymore. So I don't really see much point in keeping this parameter as it will result in error anyway